hapi/contact_lists.py

```diff
@@ -15,13 +15,28 @@ def __init__(self, *args, **kwargs):
         super(ContactListsClient, self).__init__(*args, **kwargs)
         self.log = logging_helper.get_log('hapi.contact_lists')
 
-    def _get_path(self, subpath):
-        return 'contacts/v%s/%s' % (self.options.get('version') or CONTACT_LISTS_API_VERSION, subpath)
+    def _get_path(self, method):
```

Comment:
The word "method" in a restful API may refer to "GET", "POST", "PUT", "DELETE" specifically, since "subpath" is used everywhere else, and it's also making sense, I'd suggest not changing it.